woocommerce.php

```diff
@@ -328,8 +329,9 @@ public function init() {
 
 		// Classes/actions loaded for the frontend and for ajax requests.
 		if ( $this->is_request( 'frontend' ) ) {
-			$this->cart     = new WC_Cart();                                    // Cart class, stores the cart contents
-			$this->customer = new WC_Customer();                                // Customer class, handles data such as customer location
+			$this->cart             = new WC_Cart();                              // Cart class, stores the cart contents
+			$this->customer         = new WC_Customer();                          // Customer class, handles data such as customer location
+      $this->data_structurer  = new WC_Data_Structurer();                   // Microdata class, generates and encodes a JSON-LD
```

Comment:
So any though how should I instantiate it?
Just `new WC_Data_Structurer();` on the same place and without storing it in an object?